target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+. /lib/functions/system.sh
+. /lib/functions/uci-defaults.sh
+
+ath79_setup_interfaces()
+{
+	local board="$1"
+
+	case "$board" in
+	*)
+		ucidef_set_interfaces_lan_wan "eth0" "eth1"
+		;;
+	esac
+}
+
+ath79_setup_macs()
+{
+	local board="$1"
+
+	case "$board" in
+	esac
+
+	[ -n "$lan_mac" ] && ucidef_set_interface_macaddr "lan" $lan_mac
+	[ -n "$wan_mac" ] && ucidef_set_interface_macaddr "wan" $wan_mac
+}
```

Comment:
Maybe also add
```[ -n "$label_mac" ] && ucidef_set_label_macaddr $label_mac```
here. (I do so in my split patch.)

However, I is not terribly important.

Reply:
Done